planet/Missions.ocf/Raid.ocs/System.ocg/StringTblFR.txt

```diff
@@ -0,0 +1,215 @@
+Newton113=Eh bien... oui.
+Newton114=J'ai utilisé mon épée en bois pour combattre les buissons de baies de Wompf.
+Newton115=C'est plus que n'importe lequel d'entre nous peut le montrer.
+Newton116=Alors, quelle est ta décision ? Voulez-vous nous aider à vaincre Harx ?
+Newton117=Oui
+Newton118=Excellent!
+Newton119=Commencez par marcher vers l'est en direction de Wipfville. Vous pouvez utiliser cette pelle pour vous y rendre.
+Newton120=Une fois sur place, demandez Pyrit. C'est un vieil ami à moi et devrait pouvoir vous aider.
+Newton121=Bonne chance ! Vous en aurez besoin.
+
+Newton200=Newton, j'ai construit un avion pour la contre-attaque de Harx. Mais je n'ai pas encore de carburant.
+Newton200a=Newton, j'ai des plans pour un avion pour la contre-attaque de Harx. Mais je n'ai pas encore de carburant.
+Newton201=Très bien, %s. Vous avez donc besoin de pétrole ? C'est effectivement un problème. Harx a interdit la possession de pétrole.
+Newton202=Nous ne nous soucions pas de ses règles stupides, bien sûr. Mais nous devions vendre notre pétrole pour financer la défense et la construction du château.
+Newton203=Il ne nous reste rien dans le village. Mais nous avons du pétrole une chambre cachée dans la grotte en dessous de ce village. Vous pourriez demander la clé à Mave dans la grotte à côté de la fonderie.
+
+Lara1=Nous avons besoin de plus de bois pour pouvoir au moins finir les échafaudages avant la tombée de la nuit.
+Lara2=Pourquoi tant de hâte ?
+Lara3=Nous ne saurons jamais quand <i>ils</i> pourraient revenir.
+Lara4=Qui sont-<i>ils</i>?
+Lara5=Je vous le dirai plus tard. Pouvez-vous aller chercher du bois maintenant ?
+Lara6=S'il-vous-plaît, allez chercher du bois maintenant !
+
+Lara100=Ils ne nous laisseront jamais en paix.
+
+
+Lisa1=C'est bien de voir que le château progresse enfin.
+Lisa2=Pourquoi avez-vous besoin d'un château aussi solide ?
+Lisa3=Nous avons été attaqués. Encore et encore.
+Lisa4=Attaqué ? Par qui ?
+Lisa5=Si notre malchance perdure, vous les rencontrerez assez tôt. Veuillez nous aider avec le chantier de construction maintenant.
+Lisa6=S'il-vous-plaît, aidez-vous avec le chantier.
+
+Lisa100=Veuillez faire quelque chose contre ces monstres !
+
+Lisa200=Merci beaucoup de nous aider !
+
+
+Woody1=Oh non, je me suis blessé au bras. Maintenant, je ne peux pas couper du bois et la scierie est inactive.
+Woody2=Existe-t-il un moyen de vous aider?
+Woody3=Pourriez-vous aller vers l'est (à droite) et abattre un conifère? Je m'occuperai alors du sciage.
+Woody4=D'accord. Mais je pense que votre hache est un peu grosse pour un débutant comme moi. Avez-vous une hache normale quelque part ?
+Woody5=Mes outils sont stockés près de Rocky dans la mine Ouest (à gauche) d'ici.
+Woody6=Où puis-je abattre des arbres?
+Woody7=Il suffit de travailler sur l'un des conifères à l'est (à droite) d'ici.
+
+Woody100=Comment pouvons-nous nous opposer à cela ?
+
+
+Rocky1=Salut, avez-vous une hache pour moi ?
```

Comment:
The spaces before punctuation signs are inconsistent or I do not understand the pattern. Before the question and exclamation marks there always seems to be a space, but before a full stop there is no space. Except in line 83 the space before the question mark is missing.